### PR TITLE
[feature](nereids) set wait_time for runtime filter

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/exprs/runtime_filter.cpp [hunk 1]
```diff
@@ -1174,6 +1174,9 @@ Status IRuntimeFilter::init_with_desc(const TRuntimeFilterDesc* desc, const TQue
     _has_remote_target = desc->has_remote_targets;
     _expr_order = desc->expr_order;
     _opt_remote_rf = desc->__isset.opt_remote_rf && desc->opt_remote_rf;
+    if (desc->__isset.wait_time_ms) {
+        _rf_wait_time_ms = desc->wait_time_ms;
```
Comment (morrySnow) [inline be/src/exprs/runtime_filter.cpp:1178]: what value will be assigned to `_rf_wait_time_ms` if `desc->__isset.wait_time_ms` is false?

Comment (englefly) [reply be/src/exprs/runtime_filter.cpp:1178]: it will be the wait_time in session variable

**Comment by englefly** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/processor/post/RuntimeFilterPrunerForExternalTable.java [hunk 2]
```diff
@@ -52,7 +52,8 @@ public Plan processRoot(Plan plan, CascadesContext ctx) {
                 for (int i = 0; i < rf.getTargetScans().size(); i++) {
                     PhysicalRelation scan = rf.getTargetScans().get(i);
                     if (canPrune(scan, joinAncestors)) {
-                        rfCtx.removeFilter(rf.getTargetSlots().get(i).getExprId(), (PhysicalHashJoin) join);
+                        // rfCtx.removeFilter(rf.getTargetSlots().get(i).getExprId(), (PhysicalHashJoin) join);
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/processor/post/RuntimeFilterPrunerForExternalTable.java:55]: remove this comment line

Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/processor/post/RuntimeFilterPrunerForExternalTable.java]: done

**Comment by englefly** [comment]
run buildall

**Pull request closed without merging**